sp800-63c/sec4_federation.md

```diff
@@ -52,20 +51,20 @@ In this model, a third-party sits in the middle of the transaction and communica
 
  
 
-Brokers can enable simplified technical integrations between the RP and CSP by eliminating the need for multiple point to point integrations. Brokers can also provide business confidentiality and mitigate some of the privacy risks of point to point federation described above by passing the assertions while blinding the participants on either side of the transaction to each other. For example, organizations may not wish to reveal their subscriber lists to each other and blinding prevents the capability for CSPs or RPs to track and profile subscribers. However, this model transfers this capability to the broker.  
+Brokers can enable simplified technical integrations between the RP and CSP by eliminating the need for multiple point to point integrations, which can be onerous for protocols which do not support [dynamic registration](#dynamic-registration). If implemented in very specific ways, brokers can also provide some business confidentiality and transfer some of the privacy risks of point to point federation described above by passing the assertions while blinding the participants on either side of the transaction to each other. For example, organizations may not wish to reveal their subscriber lists to each other and some implementations of blinding technology somewhat prevent the capability for CSPs or RPs to track and profile subscribers. However, this model transfers this capability to the broker.  While the use of brokers can give the appearance of complete privacy preservation, in many instances the implementation of broker technology still allows the tracking of subscribers, CSPs, and RPs in different ways.  
 
-There is a spectrum of blinding technologies that offer varying levels of privacy protection. However, as the level of blinding increases, so does the technical implementation complexity. Although privacy policies may dictate appropriate use by the CSP, RP, and the broker, blinding technology is far more effective, by making the data unavailable.
+While some broker deployments offer no additional privacy protection, some can offer limited additional levels of privacy to the subscriber through a variety of blinding technologies.  However, as the level of blinding increases, so does the technical implementation complexity. Privacy policies may dictate appropriate use by the CSP, RP, and the broker,  but blinding technology can increase effectiveness of these policies by making the data more difficult to access.
```

Comment:
While some broker deployments offer no additional privacy protection to the subscriber, others may  through a variety of blinding technologies. 

Add to however sentence "technical and operational implementation complexity."

Add at end, the following list illustrates a spectrum of blinding implementations.